activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -421,6 +421,12 @@ def drop_join_table(table_1, table_2, **options)
       #     t.column :name, :string, limit: 60
       #   end
       #
+      # ====== Change type of a column
+      #
+      #   change_table(:suppliers) do |t|
+      #     t.change :metadata, :jsonb
```

Comment:
Can you use `:json` instead of `:jsonb` which is only available for PostgreSQL?
And add `[ci skip]` for the doc only change.

Reply:
@kamipo done both.
I thought that `[ci skip]` stopped working after switching from Travis to Buildkite?

Reply:
At least `[ci skip]` is working on the commit 835cf765723892efa96ee34fa303c92c4cdcd7d6.